storage/rocksdb/ha_rocksdb.cc

```diff
@@ -60,6 +60,7 @@
 #include "./rdb_index_merge.h"
 #include "./rdb_mutex_wrapper.h"
 #include "./rdb_threads.h"
+#include "./rdb_psi.h"
```

Comment:
Can you move this up so that the header files are in alphabetical order.  We have a linter that flags this.